src/regthreesixty.js

```diff
@@ -7,7 +7,7 @@
  * # regThreesixty
```

Comment:
General hint: I know there's no comment/JSDoc in the whole document, but it would be great if you could add it, at least, to your newly added methods.